www/tynk00/cv02/foot.php

```diff
@@ -0,0 +1,60 @@
+<script>
+
+    <?php foreach ($people as $person) : ?>
+   
+
+    var cPage<?php echo $person->id ?> = 1;

```

Comment:
tak to vidim poprve ze nekdo generuje javascriptu pomoci PHP, to uz prosim nedelejte